core/src/test/java/org/elasticsearch/action/fieldstats/FieldStatsRequestTests.java

```diff
@@ -66,7 +66,7 @@ public void testFieldsParsing() throws Exception {
         assertThat(request.getIndexConstraints()[3].getComparison(), equalTo(LTE));
         assertThat(request.getIndexConstraints()[4].getField(), equalTo("field5"));
         assertThat(request.getIndexConstraints()[4].getValue(), equalTo("2"));
-        assertThat(request.getIndexConstraints()[4].getProperty(), equalTo(MAX));
+        assertThat(request.getIndexConstraints()[4].getProperty(), equalTo(MIN));
```

Comment:
For those following along at home, this is required because the example data had a duplicate key and @danielmitterdorfer switched it to `min` in the file.